package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hemilabs/smart-order-router",
-  "version": "3.26.1",
+  "version": "3.26.2",
```

Comment:
I think we were using `beta-*` versions to our forked packages for the time being. See [universal-router-sdk](https://github.com/hemilabs/universal-router-sdk/blob/0a0b4fc0de85a833c2e0a2f926edc11e5b4ae6cd/package.json#L3). However, I do see we did not that for `sdk-core` 🤔

Reply:
Changed to 3.26.1-beta.2
I know it did not say 3.26.1-beta.1 in the package.json before, but that is the published version that's used in routing-api:
<img width="509" alt="Captura de pantalla 2024-05-08 a la(s) 10 19 24" src="https://github.com/hemilabs/smart-order-router/assets/1864435/2ba7815a-7dc0-420d-8c2d-339fe0bdb7ef">

Reply:
ok, we may have updated it in a branch published but never merged it to master 🤦🏽